draft-ietf-scitt-architecture.md

```diff
@@ -447,8 +443,8 @@ In these cases, distinct Issuers (`iss`) might agree to use the `sub` CWT Claim
 
 Transparency Service:
 
-: an entity that maintains and extends the Append-only Log, and endorses its state.
-A Transparency Service can be a complex system, requiring the Transparency Service to provide many security guarantees about its Append-only Log.
+: an entity that maintains and extends the Verifiable Data Structure, and endorses its state.
+A Transparency Service can be a complex system, requiring the Transparency Service to provide many security guarantees about its Verifiable Data Structure.
```

Comment:
```suggestion
A Transparency Service can be a complex system, requiring the Transparency Service to provide many security guarantees about its Append-only Log.
```

I think the useful end guarantee is about the log and its entries, the VDS is the mechanism used to implement the guarantees.